spyder/plugins/application/container.py

```diff
@@ -297,21 +311,36 @@ def _check_updates_ready(self):
                         "<code>conda install spyder={}</code><br><br>"
                     ).format(latest_release)
                 else:
-                    content = _(
-                        "Click <a href=\"{}\">this link</a> to "
-                        "download it.<br><br>"
-                    ).format(url_r)
+                    if os.name == 'nt' and is_pynsist():
+                        box.setStandardButtons(QMessageBox.Yes |
+                                               QMessageBox.No)
+                        content = _(
+                            "You want to download and install the latest "
+                            "version of Spyder?<br><br>"
+                        )
+                    else:
+                        content = _(
+                            "Click <a href=\"{}\">this link</a> to "
+                            "download it.<br><br>"
+                        ).format(url_r)
+
                 msg = header + content + footer
                 box.setText(msg)
                 box.set_check_visible(True)
-                box.show()
+                box.exec_()
+
+                if box.result() == QMessageBox.Yes:
+                    self.application_update_status.start_installation()
+                elif(box.result() == QMessageBox.No):
+                    self.application_update_status.set_status_pending()
                 check_updates = box.is_checked()
             elif feedback:
                 msg = _("Spyder is up to date.")
                 box.setText(msg)
                 box.set_check_visible(False)
                 box.exec_()
                 check_updates = box.is_checked()
+                self.application_update_status.set_no_status()
```

Comment:
Missing `else` in case `feedback` is `False` at startup @jsbautista :
```suggestion
                self.application_update_status.set_no_status()
            else:
                self.application_update_status.set_no_status()
```